.github/workflows/client-ios.yml

```diff
@@ -63,6 +63,9 @@ jobs:
           bundle install --jobs 4 --retry 3
       - run: expotools client-build --platform ios
         timeout-minutes: 60
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

Comment:
Wouldn't it be better to add this env to the entire job instead of both steps?

Reply:
I followed the convention set by the `docs` workflow

https://github.com/expo/expo/blob/63f388bed1978890a2e57dab03c0871c6643c706/.github/workflows/docs.yml#L67-L71